contrib/platforms/macosx

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-with_blas=Apple
```

Comment:
there is a gigantic performance difference on M1. It deserves either to check ourselves for lapacke, or at least put a message about the performance implications especially in a release branch.